README.md

```diff
@@ -30,21 +30,25 @@ QChat leverages event invokation to broadcast realtime messages across the chann
 
 ## Setup
 
-### McAfee OpenDXL SDK
+### Dependencies
 
-https://www.mcafee.com/us/developers/open-dxl/index.aspx
+Qchat only supports Linux Desktop systems.
```

Comment:
The curses import has been removed and code updated to reflect. It was antiquated pre appJar implementation.